.github/workflows/int-and-connected-test-run.yml

```diff
@@ -158,138 +90,98 @@ jobs:
         ports:
           - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    env:
```

Comment:
Note for reviewers: these are required here because otherwise the shell scripts will fail.